src/LetsEncrypt/LetsEncryptOptions.cs

```diff
@@ -65,6 +66,11 @@ public bool UseStagingServer
         /// </summary>
         public X509Certificate2? FallbackCertificate { get; set; }
 
+        /// <summary>
+        /// Asymetric encryption algorithm: RS256, ES256, ES384, ES512
+        /// </summary>
+        public KeyAlgorithm KeyAlgorithm { get; set; } = KeyAlgorithm.ES256;
```

Comment:
Sorry for being picky here, but this piece seems unrelated to upgrading to ACMEv2. Git history and PRs are a little easier to follow if we keep the features in separate commits/PRs. I'm okay adding a new API for this, but let's do it separately.

Also, I'd rather not use `Certes.KeyAlgorithm` as the type of this property. I've intentionally avoided exposing Certes in the public API. In the (hopefully unlikely) event I needed to switch to a different ACME client, it makes it much easier if Certes is an implementation detail. I don't see anything in System.Security.Cryptography that's a suitable replacement, so can we add an enum of our own with the same values?

Reply:
sure let's merge the options first #25